### ISSUE TITLE
[Proposal] "Crammed"

### ISSUE THREAD

**Issue by LudwigVonChesterfield** [miki151/keeperrl#1548]
I think that if minions would feel bad about being in a room that's crammed with stuff, that would discourage just placing furniture everywhere, on every tile, and would encourage making large dungeons with corridors and stuff

**Comment by miki151** [comment]
Good idea, but the game would need to convey this to the player visually. Ideas?

**Comment by GameDoungen** [comment]
An effect maybe, that way players could tell if they clicked on a unit. Question for you, can you make moral more visible.

**Comment by LudwigVonChesterfield** [comment]
@miki151 if it was to use "zones" to determine how crammed a particular room is - then I'd probably make an option to highlight "crammed" zones - the tiles of those would appear "red".

Another possibility is of course minions "waddling" more intensively in crammed areas, perhaps sometimes falling over(?), and swear-censorship effect(#!#*!@#*!@) appearing above their head to show how displeased they are by such a predicament.
